homeassistant/components/velbus/light.py

```diff
@@ -83,22 +96,22 @@ class VelbusButtonLight(VelbusEntity, LightEntity):
     _attr_entity_registry_enabled_default = False
     _attr_supported_feature = SUPPORT_FLASH
 
-    def __init__(self, channel):
+    def __init__(self, channel: VelbusChannel) -> None:
         """Initialize the button light (led)."""
         super().__init__(channel)
         self._attr_name = f"LED {self._channel.get_name()}"
 
     @property
-    def is_on(self):
+    def is_on(self) -> Any:
```

Comment:
If we need to adjust the body to return the boolean, or cast the value to a boolean to have mypy understand it, we do that. The signature must stay in line with the defined entity interface.
```suggestion
    def is_on(self) -> bool:
```